include/session.php

```diff
@@ -30,7 +30,7 @@ function ref_session_read($id) {
 
 	$r = q("SELECT `data` FROM `session` WHERE `sid`= '%s'", dbesc($id));
 
-	if (dbm::is_result($r)) {
+	if(dbm::is_result($r)) {
```

Comment:
A bit OT: Are errors from own functions/methods being supressed? Search for this: `@$db->` to get an example.